introduction-to-datasets.md

```diff
@@ -0,0 +1,130 @@
+
+Partitioning is optional, because there are times when partitioning is not the most efficient solution. However, you should always consider partitioning as a best practice when planning your dataset.
+
+You define your partition strategy in [partition strategy JSON format][psjf] and apply it when you create your dataset. See [Partitioned Datasets][ps].
+
+When using the API, you define your dataset in the [`DatasetDescriptor.Builder`][ddb]. 
+
+For example, you can use the CLI command [`partition-config`][clipc] to define a partition strategy using name:value pairs to specify that the dataset should be partitioned on a timestamp (`ts`) field by year, month, and day.
+
+```
+user@work:~$ kite-dataset partition-config ts:year ts:month ts:day -s rating.avsc
+```
+
+The result is a partition definition in JSON format. You can use the `-o` option to out put the result and store it in a .json file.
+
+```JSON
+[ {
+  "name" : "year",
+  "source" : "ts",
+  "type" : "year"
+}, {
+  "name" : "month",
+  "source" : "ts",
+  "type" : "month"
+}, {
+  "name" : "day",
+  "source" : "ts",
+  "type" : "day"
+} ]
+```
+
+[ddb]:{{site.baseurl}}/apidocs/org/kitesdk/data/DatasetDescriptor.Builder.html#partitionStrategy(java.io.File)
+[ps]: {{site.baseurl}}/Partitioned-Datasets.html#partition-strategies
+[clipc]: {{site.baseurl}}/cli-reference.html#partition-config
+[psjf]: {{site.baseurl}}/Partition-Strategy-Format.html
+
+## Configuration Options
+
+While you need only a URI and schema to create a dataset, you can use properties for additional control over how Kite reads or writes your data.
+
+### Column Mapping
+
+HBase requires that you use column mapping to configure how your records should be stored. You define column mapping in JSON format in a data-centric way. Kite stores and retrieves the data correctly. 
+
+See [Column Mapping][cm]. See also the [CLI mapping-config command][climc].
+
+[cm]: {{site.baseurl}}/Column-Mapping.html
+[climc]: {{site.baseurl}}/cli-reference.html#mapping-config
+
+### Properties
+
+You can add custom settings to dataset properties at creation time.
+
+For example, Kite's default cache size is 10. This is the number of files that are open at any one time; each file represents a partition in the dataset written to by your application. In the CLI, you can use the `--set` option to increase the cache size of 20:
+
+```
+kite-dataset create users --schema user.avsc --set kite.writer.cache-size=20
+```
+
+You can also create custom properties for your own applications. See [Using Kite Properties][useprop].
+
+[useprop]:{{site.baseurl}}/using-kite-properties.html
+
+## Next Steps
+
+The example [Using the Kite Command Line Interface to Create a Dataset][usecli] demonstrates many of the features described above, and shows you how to create your own Hadoop dataset using only three commands.
```

Comment:
When linking to another topic, I use the full title to avoid confusion. The exception is in the TOC, where the titles are too long for the format we're using.